templates/macros/helpers.html

```diff
@@ -27,3 +27,15 @@
   } %}
 <a href="mailto:{{email}}{% if subject %}?subject={{ subject|urlencode }}{% endif %}" {{attr|xmlattr}}>{{email}}</a>
 {%- endmacro -%}
+
+{%- macro interviewer_note(interviewer_only='False', title='', type='') -%}
```

Comment:
Why is interviewer_only set as a string?

Reply:
This is gone now👍